src/sentry/utils/safe.py

```diff
@@ -118,3 +118,18 @@ def trim_dict(value, max_items=settings.SENTRY_MAX_DICTIONARY_ITEMS, **kwargs):
         if idx > max_items:
             del value[key]
     return value
+
+
+def get_path(data, path, default=None):
+    """
+    Looks up a path of properties in a nested dictionary safely.
+    Returns the value at the final level, or the default value if
+    property lookup failed at any step in the path.
+    """
+    if not isinstance(path, list) or len(path) == 0:
```

Comment:
Being pedantic, should probably also allow tuples. Can just do `isinstance(path, (list, tuple))`.

Reply:
done